app/_src/production/secure-deployment/api-server-auth.md

```diff
@@ -301,6 +301,96 @@ kumactl delete global-secret admin-user-token
    {% endtab %}
    {% endtabs %}
 
+{% if_version gte:2.2.x %}
+### Offline token issuing
+
+In addition to the regular flow of generating signing keys, storing them in secret, and using them to sign tokens on the control plane, Kuma also offers offline signing of tokens.
+In this flow, you can generate a pair of public and private keys and configure the control plane only with public keys for token verification.
+You can generate all the tokens without running the control plane.
+
+The advantages of this mode are:
+* easier, more reproducible deployments of the control plane, and more in line with GitOps.
+* potentially more secure setup, because the control plane does not have access to the private keys.
+
+Here's how to use offline issuing
+
+1. Generate a pair of signing keys
+   ```sh
+   kumactl generate signing-key --format=pem > /tmp/key-private.pem
+   kumactl generate public-key --signing-key-path=/tmp/key-private.pem > /tmp/key-public.pem
+   ```
```

Comment:
Is it worth saying that this is just creating a simple key pair and any key pair generator could make do? (Useful for vault folks).

Reply:
True, I can add that this is standard RSA key of 2048 bits